jloadr-bootstrap/src/main/java/de/adito/jloadr/bootstrap/Starter.java

```diff
@@ -37,7 +35,7 @@ public static void main(String[] args) throws Throwable
       _loadNewVersion(url);
       args[0] = url.toExternalForm();
     }
-    catch (Throwable pE)
+    catch (IOException pE)
```

Comment:
Why do you catch solely IOExceptions here? '_loadNewVersion(url)' might throw RuntimeExceptions or maybe even errors.